src/blockdata/constants.rs

```diff
@@ -100,7 +100,7 @@ pub fn genesis_block(network: Network) -> Block {
             .unwrap();
 
     match network {
-        Network::Bitcoin => {
+        Network::Prod => {
```

Comment:
We no longer need `genesis_block()` function. `bitcion_genesis_tx()` too.

Can you remove them?

Reply:
fixed at a3442bb